ansible/roles/sshd/tasks/main.yml

```diff
@@ -91,14 +91,31 @@
     mode: '0755'
   when: ansible_service_mgr != 'systemd'
 
+  # molly-guard + dracut means that the system can't reboot anymore, see:
+  # https://bugs.debian.org/992351
+- name: Ensure that dracut directory exists before installing molly-guard fix
```

Comment:
Doesn't this and the following create the directories even for users that don't use dracut?

Reply:
Yep, the problem is that if `dracut` is installed later, a reboot will also blow up...unless the workaround is already in place...